### PR TITLE
Increase the TPS of AElf.

### LINKED ISSUE #3589: Increase the TPS (Transactions Per Second) of AElf.
### Expected Behavior
The system should demonstrate significantly higher TPS (Transactions Per Second) due to improved performance optimizations.

### Specifications
As a developer, I would like to upgrade components and optimize processes to enhance the overall performance and TPS of Aelf.

**Features:**
- Upgrade .NET framework from 6 to 8.
- Parallelize block assembly and system transaction generation.
- Increase ExecutiveClearLimit from 10 to 50.
- Replace SSDB with an in-memory KV database.

**Development Tasks:**
- [ ] Upgrade .NET framework.
- [ ] Implement parallel processing for block assembly and transactions.
- [ ] Adjust ExecutiveClearLimit settings.
- [ ] Replace and configure the new in-memory KV database.

### Dependencies
nil

### References
- [Aelf TPS Improvement Documentation](https://aelfblockchain.sg.larksuite.com/wiki/YnYLwlhv8ifj6okvVD0lqAeigqe)

### PR DESCRIPTION
**Pull request by chopin-fan**
Increase the TPS (Transactions Per Second) of AElf. fixes #3589 

For detailed documentation, visit: [Aelf TPS Improvement Documentation](https://aelfblockchain.sg.larksuite.com/wiki/YnYLwlhv8ifj6okvVD0lqAeigqe).

### CODE REVIEW

File: src/AElf.Kernel.TransactionPool/Application/ITransactionPoolService.cs
Comment (chopin-fan) [inline src/AElf.Kernel.TransactionPool/Application/ITransactionPoolService.cs:1]: Adjust the code format a bit.

**Review by jason20240511 — APPROVED** [review]

File: src/AElf.CSharp.CodeOps/Validators/Whitelist/IWhitelistProvider.cs [hunk 1]
```diff
@@ -48,6 +48,8 @@ private void WhitelistAssemblies(Whitelist whitelist)
             .Assembly(typeof(IMethod).Assembly, Trust.Full) // AElf.CSharp.Core
             .Assembly(typeof(SecretSharingHelper).Assembly, Trust.Partial) // AElf.Cryptography
             .Assembly(typeof(ISmartContractBridgeContext).Assembly, Trust.Full) // AElf.Kernel.SmartContract.Shared
+            .Assembly(typeof(Groth16.Net.Verifier).Assembly, Trust.Full) // AElf.Cryptography.ECDSA
```
Comment (eanzhao) [inline src/AElf.CSharp.CodeOps/Validators/Whitelist/IWhitelistProvider.cs:51]: This is for supporting zklogin feature, it seems wired to put this upload on this PR, but not a big problem.

**Review by eanzhao — APPROVED** [review]
Nice job.

File: src/AElf.CSharp.CodeOps/Validators/Whitelist/IWhitelistProvider.cs (hunk 1 above)
Comment (chopin-fan) [reply src/AElf.CSharp.CodeOps/Validators/Whitelist/IWhitelistProvider.cs:51]: Yes, this is for supporting zklogin feature, I will roll back these codes if they cannot be verified in time.

**Pull request merged**